http_examples.cpp

```diff
@@ -57,13 +57,16 @@ int main() {
 
             string name=pt.get<string>("firstName")+" "+pt.get<string>("lastName");
 
-            *response << "HTTP/1.1 200 OK\r\nContent-Length: " << name.length() << "\r\n\r\n" << name;
+            *response << "HTTP/1.1 200 OK\r\nContent-Length: " << name.length() << "\r\n" 
```

Comment:
Thank you. Just small detail: The Content-Length parameter is usually at the end of the header is it not? Maybe place Content-Type above Access-Control-Allow-Origin as well?